src/commands/external.rs

```diff
@@ -49,10 +50,20 @@ pub async fn execute_external_subcommand(
             }
         }
         Err(Error::NotFound(e)) => {
-            tracing::debug!("Tried to resolve {plugin_name} to plugin, got {e}");
-            terminal::error!("'{plugin_name}' is not a known Spin command. See spin --help.\n");
-            print_similar_commands(app, &plugin_name);
-            process::exit(2);
+            if plugin_name == "cloud" {
+                println!("The `cloud` plugin is required. Installing now.");
```

Comment:
Suggest `eprintln!` so that if the user is capturing plugin output for _waves hands_ reasons then the message doesn't get mixed up with it.  (E.g. imagine a hypothetical `cloud variables list --output json >vars.json`.)

Or we could define a `terminal::info!` and use that (I did one for the badger though might be too dark).

Reply:
Good point. The "plugin installed" message still goes to stdout, though:
```
Plugin 'cloud' was installed successfully!

Description:
	The Fermyon Cloud Plugin

Homepage:
	https://github.com/fermyon/cloud-plugin
host
beep
```

Reply:
Ah, no worries then; we can leave it for now and revisit both messages in a later pass.  It's not a big deal - realistically the command that triggers the install will be `spin cloud login` so who cares about what stream the output is on.

(We could also look at suppressing the description/homepage announcement for 'magic' plugins - the user has not asked for an install and doesn't really need to be told about the plugin - "I just want to log in, stop _talking_ at me" kind of feeling - but again not a blocker.)

Reply:
I like that idea of conditional suppression of magic plugin info